### DISCUSSION TITLE
Is there a way to prompt a specific document?

### DISCUSSION THREAD

**Discussion by silveranalytics** [PromtEngineer/localGPT#665] [Q&A]
I have to delete the DB folder and only ingest one document at a time for the model to pick the correct source document. Is there a way to prompt it for the specific file?

**Comment by lavericklavericklaverick** [comment]
id suggest you'd need multi agent or just a search script, you can easily automate the creation of seperate dbs for each book, then another to find select that db and put it into the db folder, then run the localGPT. 
ask chatGPT to make it if you cant be bothered.
